CMakeLists.txt

```diff
@@ -11,11 +11,17 @@ FetchContent_Declare(
 	GIT_REPOSITORY https://github.com/nlohmann/json.git
 	GIT_TAG bc889afb4c5bf1c0d8ee29ef35eaaf4c8bef8a5d # release/3.11.2
 )
+FetchContent_Declare(
+	logger
+	GIT_REPOSITORY https://github.com/temikfart/logger.git
+	GIT_TAG e1e6d1122815b3b6b173e361bac0f3be5d4d2a6c
```

Comment:
change hash to the [latest version](https://github.com/temikfart/logger/releases/tag/v1.0.0) (v1.0.0).

Reply:
Don't forget to add comment like at line 12:
```cmake
GIT_TAG abcd # release/1.0.0
```